easybuild/easyconfigs/o/OpenCV/OpenCV-4.5.1-foss-2020b-contrib.eb

```diff
@@ -50,7 +52,18 @@ dependencies = [
     ('HDF5', '1.10.7'),  # needed by hdf from contrib
 ]
 
-configopts = '-DOPENCV_EXTRA_MODULES_PATH=%(builddir)s/%(namelower)s_contrib-%(version)s/modules'
+configopts = '-DOPENCV_EXTRA_MODULES_PATH=%(builddir)s/%(namelower)s_contrib-%(version)s/modules '
+configopts += '-DFREETYPE_FOUND=ON '
+configopts += '-DFREETYPE_INCLUDE_DIRS=$EBROOTFREETYPE/include/freetype2/ '
+configopts += '-DFREETYPE_LIBRARIES=$EBROOTFREETYPE/lib64/libfreetype.so '
+configopts += '-DFREETYPE_LINK_LIBRARIES=$EBROOTFREETYPE/lib64/libfreetype.so '
+configopts += '-DFREETYPE_LINK_LIBRARIES_XXXXX=ON '
+configopts += '-DHARFBUZZ_FOUND=ON '
+configopts += '-DHARFBUZZ_INCLUDE_DIRS=$EBROOTHARFBUZZ/include/harfbuzz '
+configopts += '-DHARFBUZZ_LIBRARIES=$EBROOTHARFBUZZ/lib64/libharfbuzz.so '
+configopts += '-DHARFBUZZ_LINK_LIBRARIES=$EBROOTHARFBUZZ/lib64/libharfbuzz.so '
+configopts += '-DHARFBUZZ_LINK_LIBRARIES_XXXXX=ON '
```

Comment:
What's this `XXXXX` configure option about?

Reply:
That's totally weird code in OpenCV to prevent configure from wiping <PKG>_LINK_LIBRARIES: https://github.com/opencv/opencv/blob/6e8daaec0f46aaba9ea22e2afce47307b1dbff9f/cmake/OpenCVUtils.cmake#L861
It seems they put it there as a fix to prevent exactly what's happening, ocv_check_modules not being able to recognize freetype and harfbuzz...

Reply:
I think it's worth adding a comment in the easyconfig file above this line to clarify that, because it looks really weird...

Reply:
Comment added (finally!)